modules/consentManagement.js

```diff
@@ -376,6 +376,7 @@ export function setConsentConfig(config) {
   }
   if (!addedConsentHook) {
     $$PREBID_GLOBAL$$.requestBids.before(requestBidsHook, 50);
+    if ($$PREBID_GLOBAL$$.liveConnectHook) $$PREBID_GLOBAL$$.liveConnectHook.before(requestBidsHook, 50);
```

Comment:
Please explain this -- why should the ConsentManagement module be related to LiveConnect?

Reply:
we want to setting our cookie if the consent is not given